jsonschema-validation.xml

```diff
@@ -936,42 +936,22 @@
                 <t>
                     Due to security and performance concerns, as well as the open-ended nature of
                     possible content types, implementations MUST NOT automatically decode, parse,
-                    and/or validate the string contents by default.  This additionally supports
-                    the use case of embedded documents intended for processing by a different
-                    consumer than that which processed the containing document.
+                    and/or validate the string contents.  Applications are expected to use these
+                    annotations to invoke the appropriate libraries (including JSON Schema for
+                    any further schema-based validation) separately.
```

Comment:
I don't understand what this parenthetical is saying.

Reply:
It's referring to treating the JSON Schema implementation as the "application" when you get a `contentSchema` and (after decoding and parsing whatever was in the string according to `contentEncoding` and/or `contentMediaType`) passing it back in to validate the string contents.

Let me know if you have any ideas on better wording, or if you think I should just drop the parenthetical aside.

Reply:
I see. I think it can be dropped.

Reply:
Cool- done!